app/lib/submission_builder/ty2022/states/ny/documents/state1099_g.rb

```diff
@@ -36,10 +36,10 @@ def document
                   xml.StateAbbreviationCd "NY"
                   xml.ZIPCd form1099g.recipient_zip if form1099g.recipient_zip.present?
                 end
-                xml.UnemploymentCompensation form1099g.unemployment_compensation if form1099g.unemployment_compensation.present?
-                xml.FederalTaxWithheld form1099g.federal_income_tax_withheld if form1099g.federal_income_tax_withheld.present?
+                xml.UnemploymentCompensation form1099g.unemployment_compensation_amount.round.to_i if form1099g.unemployment_compensation_amount.present?
```

Comment:
do we need both `round` and `to_i`?

Reply:
nope i'll just keep it a round totally missed this!